### PR TITLE
Feature/background video pattern#218

### PR DESCRIPTION
**Pull request by Henrikschytze**
Towards #218

### CODE REVIEW

**Comment by Dan9boi** [comment]
@Henrikschytze what's the status on this?

File: patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    height: 100vh;
+    z-index: -1;
+    pointer-events: none;
+    overflow: hidden;
+    background-size: cover;
+
+    & iframe {
+        width: 100vw;
+        height: 56.25vw;
+        /* Given a 16:9 aspect ratio, 9/16*100 = 56.25 */
```
Comment (Dan9boi) [inline patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.css]: Great job with the comments 💯

File: patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    &__foreground {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 100vw;
+        height: 100vh;
+        font-size: 1.7em;
+        color: whitesmoke;
```
Comment (Dan9boi) [inline patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.css]: Does it make sense to make hardcoded colors?

File: patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.hbs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<div class="pagheeader-video background-fullscreen" style="background-image: url({{ path '{{backgroundImagePath}}' }} )">
+  {{#if youtubeid}}
+  <div class="youtube__iframe-wrapper" data-youtubeid="{{youtubeid}}">
```
Comment (Dan9boi) [inline patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.hbs]: missing indentation?

File: patterns/03-organisms/pageheaders/pageheader-video/pageheader-video.css [hunk 4]
```diff
@@ -16,10 +16,14 @@
     overflow: hidden;
     background-size: cover;
 
-    @media (--viewport-sm-min) {
+    @media (--viewport-lg-min) {
         height: var(--video-height);
     }
 
+    &@media (--viewport-md-max) {
```
Comment (Dan9boi) [inline patterns/03-organisms/pageheaders/pageheader-video/pageheader-video.css]: is this an error?

File: patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.css (hunk 2 above)
Comment (Henrikschytze) [reply patterns/03-organisms/pageheaders/pageheader-video-fullscreen/pageheader-video-fullscreen.css]: Not a requirement for the feature, I just wanted to showcase that you can put content on top of the video as an example

**Comment by daniel-doktor** [comment]
I have looked at the examples – and we are getting there – good job until now Henrik!

I have a few comments:
- All three examples seems a bit slow – have we checked the performance?
- When browser jerking I can see the placeholder image in the back – and there is a bit of delay before the video fills out the browser again.
- Can we do something about the displaying of the placeholder image in the beginning of the video? Placeholder images should only show if the video can't play.
- The youtube video shows recommended videos in the end and doesn't loop – Christian Færch suggests to use this to the embed URL: &loop=1&playlist=VIDEOID . This will create a playlist with only this video and loop this.

**Comment by Henrikschytze** [comment]
@Dan9boi Most recent changes are in the request now

**Comment by Henrikschytze** [comment]
@Dan9boi Har fikset merge conflicts

**Pull request merged**